### ISSUE TITLE
bcftools isec: output records from both files

### ISSUE THREAD

**Issue by TaurVil** [samtools/bcftools#1641]
Hello, I'm trying to use isec to output shared sites for 2 vcf files, and I want one file with those sites in all individuals. I'm trying to do this using the following command: `bcftools isec -n=2 -c none -w 1 -w 2 -O z -o output.vcf.gz ./file1.vcf.gz ./file2.vcf.gz`, but the output only contains the genotype calls in one of the files. Is it possible to integrate isec and merge into a single command?

**Comment by pd3** [comment]
The `-w` option is not used correctly, it can be given only once, e.g. `-w 1,2`. This will indeed create two files and if you want to merge them, you can follow with `bcftools merge`. There is no plan to integrate `isec` and `merge`, as that can be achieved with a simple script.

**Comment by nmunyengwa** [comment]
Hi pd3. How can I change the order of chromosomes in a vcf file.
